Update ZXC to fully follow ERC20 standard

lknix opened the pull request.

lknix committed:
Follow the latest ERC20 standard interface

lknix committed:
Conform to 0xcert style guidelines

lknix committed:
Add additional tests

lknix committed:
Use const instead of let

xpepermint reviewed (changes requested):
do not reimplement erc20 but use @0xcert/ethereum-erc20 implementation instead

lknix committed:
Switch to relative imports

lknix committed:
Switch from public functions to external
We're not calling them from inside a contract, thus there's no reason to
leave them public.

lknix committed:
Make Zxc inherit from ERC20 Token implementation

lknix committed:
Update package-log.json

xpepermint commented on contracts/tokens/Zxc.sol:
```diff
@@ -94,26 +94,70 @@ contract Zxc is Ownable {
 
-    balances[owner] = currentSupply;
-    emit Transfer(address(0x0), owner, currentSupply);
+    balances[owner] = tokenTotalSupply;
```
We must not forget to change the owner address to 0xcert master wallet.
